packages/auth/src/AuthService.ts

```diff
@@ -0,0 +1,44 @@
+import Keycloak from "keycloak-js";
+
+export class AuthService{
+    static auth: any = {};
+
+    loadUserProfile(): any {
```

Comment:
I think we should create a type for the return `UserProfile` to make it easier for developers to work with, and provide documentations.

Reply:
https://github.com/keycloak/keycloak-js-bower/blob/master/dist/keycloak.d.ts#L181-L191

Reply:
Any is bad. If we do wrapper let's not make it worse than official lib:

Reply:
Any is bad. If we do wrapper let's not make it worse than official lib:
https://github.com/keycloak/keycloak-js-bower/blob/master/dist/keycloak.d.ts#L476

Here is user interface that @wei-lee is mentioning:
https://github.com/keycloak/keycloak-js-bower/blob/master/dist/keycloak.d.ts#L181-L191